OpenRA.Game/Network/OrderManager.cs

```diff
@@ -28,11 +28,10 @@ public sealed class OrderManager : IDisposable
 		public Session.Client LocalClient { get { return LobbyInfo.ClientWithIndex(Connection.LocalClientId); } }
 		public World World;
 
-		public readonly string Host;
-		public readonly int Port;
+		public readonly ConnectionTarget Endpoint;
 		public readonly string Password = "";
 
-		public string ServerError = "Server is not responding";
+		public string ServerError = null;
```

Comment:
why the null here?

Reply:
This enables the connection error dialog to fall back to a message set on the connection object if the server does not specifically override it. If neither are set, the connection dialog will fall back to "Unknown error".

Reply:
thanks for the quick response.